### PR TITLE
Adds non-browser-based test script

### PR DESCRIPTION
**Pull request by cuducos**
This PR suggests a simple script to call CKAN JavaScript Client and create and push a blob to a resource using CKAN API. In the future, it can be moved to Cypress, but at the moment I am struggling to run async code inside Cypress runner, so this comes in as a _hot fix_ to be reverted soon.

### CODE REVIEW

**Comment by rufuspollock** [comment]
@cuducos great and I assume you have run this and it works (?).

File: README.md [hunk 1]
```diff
@@ -177,4 +177,16 @@ As mentioned below in best practices please _make sure your tests are always iso
...
-- You can add the constants and files on `cypress/fixtures` — for example, the `cypress/fixtures/sample.csv` which can be used in commands and tests.
\ No newline at end of file
+- You can add the constants and files on `cypress/fixtures` — for example, the `cypress/fixtures/sample.csv` which can be used in commands and tests.
+
+#### Non-browser based integration test
+
+There is also a script to use [CKAN JavaScript client](https://github.com/datopian/ckan-client-js) to run in integration test:
+
+```console
+$ node ntegration_test_with_ckan_client.js <ORGANIZATION NAME>
```
Comment (rufuspollock) [inline README.md:187]: is that a typo: "ntegration_test..."

File: README.md [hunk 2]
```diff
@@ -177,4 +177,16 @@ As mentioned below in best practices please _make sure your tests are always iso
...
+```
+
+This script requires a CKAN instance set with an organization.
```
Comment (rufuspollock) [inline README.md:190]: What does this script do? I assume it auto creates a dataset, resource and then uploads file. Let's mention that and make sure his information it output by the script (e.g. urls are output to dataset and resource).

**Review by rufuspollock — CHANGES_REQUESTED** [review]
Small questions.

**Comment by cuducos** [comment]
Thanks for the feedback. I think @cotts, @kmanaseryan and I can quickly work on #9 and #10, making this PR obsolete.Let's keep it open more 24h and we decide to close or merge this. Does that make sense?

**Comment by cuducos** [comment]
Closed in favor of #13 (although documentation as described in #9 is still pending).

**Pull request closed without merging**